src/BlockEntities/ChestEntity.cpp

```diff
@@ -38,7 +40,11 @@ void cChestEntity::SendTo(cClientHandle & a_Client)
 	// The chest entity doesn't need anything sent to the client when it's created / gets in the viewdistance
 	// All the actual handling is in the cWindow UI code that gets called when the chest is rclked
 
-	UNUSED(a_Client);
+	// Temporary fix for invisible chests (#3479, #3403)
+	if (a_Client.GetProtocolVersion() >= cProtocolRecognizer::PROTO_VERSION_1_9_0) 
+	{
+		a_Client.SendUpdateBlockEntity(*this);
+	}
```

Comment:
Actually I was looking here http://wiki.vg/Protocol in "Update Block Entity" section. There's no action related with chests there. 
According to the link you posted I have to check if chest is open to send "1" followed by the number of players looking at. Or 0 if it's closed. But in this context action field "1" means "Set data of a mob spawner".  Anyway now is sending "0", that means chest closed .